code/modules/mob/living/say.dm

```diff
@@ -255,7 +255,10 @@ proc/get_radio_key_from_channel(var/channel)
 		if(M.client)
 			speech_bubble_recipients.Add(M.client)
 	spawn(0)
-		flick_overlay(image('icons/mob/talk.dmi', src, "h[speech_bubble_test]",MOB_LAYER+1), speech_bubble_recipients, 30)
+		if(istype(src.loc, /obj/mecha))

```

Comment:
I'm 90% sure you two are confusing each other
@RemieRichards is trying to propose, instead of a god awful snowflake istype() check in saycode, there should be an atom-level proc that does nothing by default, then, if subtypes want to do something (mecha, in this instance), they override it